Mendelugram/Mendelugram/View/Comment/CommentTableViewCell.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  CommentTableViewCell.swift
+//  Mendelugram
+//
+//  Created by Pacek on 21/04/2019.
+//
+
+import UIKit
+
+class CommentTableViewCell: UITableViewCell {
+
+    static let reuseIdentifier = String(describing: CommentTableViewCell.self)
+
+    struct Input {
+        let avatar: UIImage
+        let username: String
+        let comment: String
+    }
+
+    @IBOutlet weak var avatarImageView: UIImageView!
+    @IBOutlet weak var commentLabel: UILabel!
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        avatarImageView.layer.cornerRadius = avatarImageView.frame.width / 2
```

Comment:
Can you move this into a setup or configure method?

Reply:
I don't see a point in moving a one line that should happen in awakeFromNib a to special configure() method that will be called from awakeFromNib anyway.

Reply:
You have two lines there :-) And from my point of view, it is nice to keep only one approach throughout a whole codebase i.e. when you setup anything either from `viewDidLoad` or from `awakeFromNib` it is always in a method called `setup`.